src/cli/args.zig

```diff
@@ -212,7 +215,7 @@ fn formatValues(comptime T: type, key: []const u8, writer: anytype) std.mem.Allo
         if (std.mem.eql(u8, key, f.name)) {
             switch (@typeInfo(f.type)) {
                 .@"enum" => |e| {
-                    try writer.print(", valid values are: ", .{});
+                    try writer.writeAll(std.mem.span(i18n._(", valid values are: ")));
```

Comment:
This entire message needs to be rewritten so that it flows as one entire sentence